src/functions/getResult/framework/handler.ts

```diff
@@ -33,7 +33,7 @@ export async function handler(event: APIGatewayEvent, fnCtx: Context): Promise<R
 
     const result: TestResultRecord[] = await getResult(appRefPathParam);
 
-    const results: StandardCarTestCATBSchema[] = result.map(row => row.test_result);
+    const results: CatBUniqueTypes.TestResult[] = result.map(row => row.test_result);
```

Comment:
Should we not be using the union type here rather than coupling to cat B? Union type is called `TestResultSchemasUnion` and resides here - https://github.com/dvsa/mes-api-definitions/blob/develop/mes-test-schema/categories/index.d.ts

Reply:
Absolutely right. Changed to use TestSchemasUnion type.